### ISSUE TITLE
Issues with keyboard navigation in messages list

### ISSUE THREAD

**Issue by WofWca** [deltachat/deltachat-desktop#4480]
- Operating System (Linux/Mac/Windows/iOS/Android): All
- Delta Chat Version: 293a665fbf1236a90776bb25e5f3154453a88805 (pre-1.51)

We've just merged https://github.com/deltachat/deltachat-desktop/pull/4294, which IMO is big improvement, but it's not bug-free and not completely nice in general. See the TODO items in that MR.

**Comment by nicodh** [comment]
Is this a bug or more an improvement?
Are there any plans to work on that? (it's more than one year old). If not maybe close it

**Comment by WofWca** [comment]
Well those issues mentioned in that MR are not super important, but it's also hard to call them "feature requests".

No, I have no concrete plans on working on this (but not to say that it shouldn't be worked on).
The most important one I guess was the "react to message doesn't work", but now it does since not so long ago.
